### DISCUSSION TITLE
how do I trigger blockscout to reindex?

### DISCUSSION THREAD

**Discussion by hmvien** [blockscout/blockscout#10189] [Q&A] [answered]
Hi, I have my own blockscout running indexing a local app chain. I would like to trigger reindexing. Is there a way to do that without dropping my database and re-creating?

**Comment by sl1depengwyn** [comment, marked as answer]
Add block ranges that you want to reindex to `missing_block_ranges` table where `from_number` >= `to_number`, but be careful, after insert there should be no intersecting ranges

**Reply by hmvien** [reply]
This was it, thank you!

**Comment by hmvien** [comment]
@sl1depengwyn I tried to get it to reindex from the current latest block e.g. 1108372 to the first block, but I'm not receiving signal that it ever got back from the beginning. The `id` is monotonically increasing and it is only at 309177, suggesting it never reindexed more than 300k blocks. 

Am I doing this right?

**Reply by sl1depengwyn** [reply]
Hm, please tell which version of Blockscout do you run?
